website/docs/r/storage_bucket_iam_policy.html.markdown

```diff
@@ -0,0 +1,58 @@
+---
```

Comment:
Looks like you've duplicated the bucket docs here?

Reply:
No; Why?

Reply:
Looks like you have both a `google_storage_bucket_iam_policy.html.markdown` and a nearly identical `storage_bucket_iam_policy.html.markdown` file (without the "google" prefix). I assume that's what @paddycarver is referring to.

Reply:
Yup, sorry. I'm seeing two different files here with almost the same name and can't tell what happened.

Reply:
FYI, @sebglon deleted the duplicate file. I think the only outstanding issue left is the "this part of the code scares me" section, if you can expand on that?